packages/components/src/components/post-tab-header/post-tab-header.tsx

```diff
@@ -32,17 +32,15 @@ export class PostTabHeader {
 
   render() {
     return (
-      <Host data-version={version}>
-        <button
-          aria-selected="false"
-          class="tab-title"
-          id={this.tabId}
-          role="tab"
-          tabindex="-1"
-          type="button"
-        >
-          <slot />
-        </button>
+      <Host
+        id={this.tabId}
+        role="tab"
+        data-version={version}
+        aria-selected="false"
+        tabindex="-1"
+        class="tab-title"
+      >
+        <slot />
```

Comment:
Button had to be removed because of https://dequeuniversity.com/rules/axe/4.7/nested-interactive?application=axeAPI